server/mailet/mailets/src/main/java/org/apache/james/transport/mailets/SPF.java

```diff
@@ -124,7 +124,7 @@ private void createSPF(org.apache.james.jspf.core.DNSService dnsProbe) {
         MacroExpand macroExpand = new MacroExpand(dnsProbe);
         wiringService.put(MacroExpandEnabled.class, macroExpand);
         RFC4408SPF1Parser parser = new RFC4408SPF1Parser(new DefaultTermsFactory(wiringService));
-        SynchronousSPFExecutor executor = new SynchronousSPFExecutor(dnsProbe);
+        AsynchronousSPFExecutor executor = new AsynchronousSPFExecutor(dnsProbe);
```

Comment:
I think we just need to revert code of commit: 69742a87cfc5d0d9048246968e2b58677b852d59 (pr https://github.com/apache/james-project/pull/1627)

It will avoid redundant code

Reply:
IMO not worth the pain...